contracts/scripts/test/measure_gas.ts

```diff
@@ -16,7 +16,7 @@ async function main() {
 }
 
 async function logGasForTransfers() {
-  const transferCounts = [31];
+  const transferCounts = [241, 242, 243, 244, 245, 246, 247, 248, 249, 250];
```

Comment:
@jacque006 Update this to more staggered numbers. Maybe add failure row if txn fails/is rejected.